content/resources/chapter-2/mining.tsx

```diff
@@ -0,0 +1,27 @@
+'use client'
+
+import { useTranslations } from 'hooks'
+
+import { Text, ResourcePage } from 'ui'
+
+export default function MiningResources({ lang }) {
+  const t = useTranslations(lang)
+
+  return (
```

Comment:
Is it intentional for this help page to be so small?

Reply:
Just not sure what to put there yet since it's just a click-through challenge..